<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>facebookBot</class>
 <widget class="QDialog" name="facebookBot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Facebook Message Bot</string>
  </property>
  <widget class="QTextEdit" name="userNameInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="userNameLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>UserName:</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="messageInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Message:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="updateText">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="messageDelayTimeInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="messageDelayTimeLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Message Delay Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="findNextFriendsDelayLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Next Friend Delay Time:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="findNextFriendsDelayInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
